aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -100,6 +100,11 @@ module aptos_framework::fungible_asset {
         project_uri: String,
     }
 
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+    struct GlobalFreeze has key {
```

Comment:
freeze has a different meaning. 

Let's call this differently / more descriptively, and not share error codes.

maybe DelegatedToOverloaded? ActionsOverloaded?
also, can only one of the withdraw/deposit be overloaded, or do both need to be simultanously? or if either is, we just ask them to go through the other api anyways?